superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -699,21 +699,21 @@ export const controls = {
     freeForm: true,
     label: t('Time Granularity'),
     default: 'one day',
-    choices: formatSelectOptions([
-      'all',
-      '5 seconds',
-      '30 seconds',
-      '1 minute',
-      '5 minutes',
-      '1 hour',
-      '6 hour',
-      '1 day',
-      '7 days',
-      'week',
-      'week_starting_sunday',
-      'week_ending_saturday',
-      'month',
-    ]),
+    choices: [
+      [null, 'all'],
+      ['PT5S', '5 seconds'],
```

Comment:
If a chart has been saved with a string say `5 seconds` as a value, did you confirm that this load up ok in here as a "free form" value? From my understanding it should.

Reply:
@betodealmeida ^ please confirm and I'll merge

Reply:
It does, it loads with the saved value as freeform:

<img width="262" alt="screen shot 2018-04-06 at 11 16 44 am" src="https://user-images.githubusercontent.com/1534870/38437359-0cd5cc42-398c-11e8-865b-32c646d21699.png">